python/ray/dashboard/client/src/pages/dashboard/logical-view/Actor.tsx

```diff
@@ -137,16 +163,18 @@ class Actor extends React.Component<Props & WithStyles<typeof styles>, State> {
             },
             {
               label: "State",
-              value: actor.state.toLocaleString(),
+              // If the state is alive, there's no need to propogate that to user.
+              value: actor.state === 0 ? "" : ActorStateHumanNames[actor.state],
             },
             {
               label: "Resources",
               value:
-                Object.entries(actor.usedResources).length > 0 &&
-                Object.entries(actor.usedResources)
-                  .sort((a, b) => a[0].localeCompare(b[0]))
-                  .map(([key, value]) => `${value.toLocaleString()} ${key}`)
-                  .join(", "),
+                Object.entries(actor.usedResources).length > 0
+                  ? Object.entries(actor.usedResources)
+                      .sort((a, b) => a[0].localeCompare(b[0]))
+                      .map(([key, value]) => `${value.toLocaleString()} ${key}`)
+                      .join(", ")
+                  : "",
```

Comment:
Maybe it would make sense to use `null` for items we don't wish to display? (Same goes for the other places where we're using an empty string.)